modules/reference/pages/systems/bootstrapping.adoc

```diff
@@ -30,6 +42,7 @@ Disable SSH Strict Key Host Checking::
 This check box is selected by default.
 This allows the script to auto-accept host keys without requiring a user to manually authenticate.
 
+// Aside from the terrible formatting here, does this need the tech preview notice removed? --LKB 2020-04-17
```

Comment:
Wow, I didn't know that we still have "minion-less" salt clients as a tech preview. Is this true @renner ?

Reply:
I suspect it's more that we failed to take the note out some time ago, honestly ...